akka-cluster/src/main/scala/akka/cluster/routing/ClusterRouterConfig.scala

```diff
@@ -46,33 +55,46 @@ final case class ClusterRouterGroupSettings(
   totalInstances:    Int,
   routeesPaths:      immutable.Seq[String],
   allowLocalRoutees: Boolean,
-  useRole:           Option[String]) extends ClusterRouterSettingsBase {
+  useRoles:          Set[String]) extends ClusterRouterSettingsBase {
```

Comment:
Ok, I will add copy methods with previous signature and see if it now passes the binary compatibility check.